lib/genres.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'movie.dart';
+
+class Genres extends StatelessWidget {
+  final Movie _mov;
```

Comment:
Minor: always prefer complete names rather than contractions or abbreviations. `_movie` in this case should be suitable!

Reply:
done